hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/CopyContext.java

```diff
@@ -0,0 +1,34 @@
+package org.apache.hadoop.fs.s3a;
+
+import com.amazonaws.services.s3.transfer.Copy;
+
+class CopyContext {
+
+  private final Copy copy;
+  private final String srcKey;
+  private final String dstKey;
+  private final long length;
+
+  CopyContext(Copy copy, String srcKey, String dstKey, long length) {
+    this.copy = copy;
+    this.srcKey = srcKey;
+    this.dstKey = dstKey;
+    this.length = length;
+  }
+
+  Copy getCopy() {
+    return copy;
+  }
+
+  String getSrcKey() {
+    return srcKey;
+  }
+
+  String getDstKey() {
```

Comment:
no, go on, use "destKey". We can afford the letter e's

Reply:
Fixed